engine/overworld/player_movement.asm

```diff
@@ -665,7 +665,7 @@ ENDM
 
 	ld hl, OBJECT_FLAGS2
 	add hl, bc
-	set 2, [hl]
+	set OBJ_FLAGS2_2, [hl]
```

Comment:
Not sure what this is for.

Reply:
Looks to me like a boulder moving flag... 

Its set here if the object the player is running into is a boulder.
Its checked in MovementFunction_Strength to determine if the function should move the boulder or not.
its reset in the StepFunction_StrengthBoulder when the movement is complete.

So maybe... `BOULDER_MOVING_F`?